iOS/Flows/Conversation/Cells/MessageCell.swift

```diff
@@ -11,17 +11,25 @@ import SwiftUI
 import StreamChat
 import Combine
 
+protocol MesssageCellDelegate: AnyObject {
+    func messageCell(_ cell: MessageCell, didTapMessage messageInfo: (ConversationId, MessageId))
+    func messageCell(_ cell: MessageCell, didTapEditMessage messageInfo: (ConversationId, MessageId))
+    func messageCell(_ cell: MessageCell, didTapAttachment attachment: MediaItem)
+}
```

Comment:
There was starting to be enough input events that I thought it best to move to a delegate pattern.